lib/compat.php

```diff
@@ -44,6 +44,20 @@ function register_block_type_from_metadata( $path, $args = array() ) {
 	}
 }
 
+/**
+ * Extends block editor settings to determine whether to use drop cap feature.
+ *
+ * @param array $settings Default editor settings.
+ *
+ * @return array Filtered editor settings.
+ */
+function gutenberg_extend_settings_drop_cap( $settings ) {
+	$settings['__experimentalDisableDropCap'] = get_theme_support( 'disable-drop-cap' );
```

Comment:
I think we should stop adding theme supports, instead I'd prefer if we'd stabilize theme.json in a release or two, we need to give it more time to be tested by themes. Potentially it can make it into 5.5

Reply:
Makes sense, I will set it to `false` for now.